graphql/schema/types/stash-box.graphql

```diff
@@ -0,0 +1,32 @@
+type StashBox {
+    endpoint: String!
+    api_key: String!
+}
+
+input StashBoxInput {
+    endpoint: String!
+    api_key: String!
+}
+
+type SceneStashID {
+  scene_id: ID!
+  endpoint: String!
+  stash_id: String!
+}
+
+type PerformerStashID {
+  performer_id: ID!
+  endpoint: String!
+  stash_id: String!
+}
+
+type StudioStashID {
+  studio_id: ID!
+  endpoint: String!
+  stash_id: String!
+}
```

Comment:
I think these should all be a single type `StashID` and remove the id. The instances of these are all within the scope of their containing objects, so they are redundant anyway.

Reply:
Good point, fixed.